Sources/NIOFileSystem/FileSystem.swift

```diff
@@ -393,6 +396,42 @@ public struct FileSystem: Sendable, FileSystemProtocol {
     public func removeItem(
         at path: FilePath,
         recursively removeItemRecursively: Bool
+    ) async throws -> Int {
+        try await self.removeItem(at: path, strategy: .platformDefault, recursively: removeItemRecursively)
+    }
```

Comment:
Breadcrumb: I added this believing we need it to not break the API. We now think this might just be the checker that's overly sensitive and throws a false-positive at us.